# Generated by Django 3.1.1 on 2020-10-23 06:47

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('account', '0009_auto_20201023_1529'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='email',
            field=models.EmailField(error_messages={'unique': 'そのEメールアドレスはすでに使用されています'}, help_text='150文字以下。文字と数字と @/./+/-/_ のみ。', max_length=254, unique=True, verbose_name='Eメール'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(max_length=150, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='ユーザ名'),
        ),
    ]
